### PR TITLE
Fix zoho push lead

### PR DESCRIPTION
**Pull request by Dcoutelle**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Automated tests included? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/3651 #5485
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
The zoho plugin can not push contacts to zoho due to an incorrect function name

#### Steps to test this PR:
1. see issue https://github.com/mautic/mautic/issues/3651
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.

### CODE REVIEW

File: plugins/MauticCrmBundle/Integration/CrmAbstractIntegration.php [hunk 1]
```diff
@@ -187,6 +187,18 @@ public function getApiHelper()
...
     }
 
+    /**
+     * Sets the API helper.
+     *
+     * @param object $helper
+     */
+    public function setApiHelper($helper)
+    {
+        if ($helper instanceof CrmApi) {
```
Comment (alanhartless) [inline plugins/MauticCrmBundle/Integration/CrmAbstractIntegration.php]: Is there a reason for this over type hinting $helper? Also, public setters used only for tests can be avoided by using partial mocks using setConstructorArgs then setMethods or setMethodsExcept(). 

`

**Comment by robwent** [comment]
I've tested this as an action on a form and also in a campaign push to integration action and I get no errors.

Contacts get added to Zoho crm in the 'leads' area.

I haven't tested the `php /home/path/to/app/console mautic:integration:fetchleads -i Zoho` command to sync everything

**Comment by yourdigitalclub** [comment]
I receive the following error

`[2018-01-12 11:35:53] mautic.WARNING: PHP Warning - Invalid argument supplied for foreach() - in file /home/mautbabington/public_html/app/bundles/PluginBundle/Integration/AbstractIntegration.php - at line 1853 [] []`

**Comment by guizero** [comment]
Could anybody take a look at this PR?

**Pull request merged**